cmake/modules/hunter_pkgconfig_export_target.cmake

```diff
@@ -51,37 +46,45 @@ function(hunter_pkgconfig_export_target PKG_CONFIG_MODULE)
   set(link_libs)
 
   hunter_status_debug(
-      "PKG_CONFIG_MODULE ${PKG_CONFIG_MODULE} LDFLAGS: ${${PKG_CONFIG_MODULE}_LDFLAGS}"
-  )
-  string(COMPARE NOTEQUAL
-      "${${PKG_CONFIG_MODULE}_LDFLAGS}"
-      ""
-      has_ldflags
+      "PKG_CONFIG_MODULE ${PKG_CONFIG_MODULE} LINK_LIBRARIES: ${${PKG_CONFIG_MODULE}_LINK_LIBRARIES}"
   )
-  if(has_ldflags)
-    list(APPEND link_libs ${${PKG_CONFIG_MODULE}_LDFLAGS})
+  if(NOT "${${PKG_CONFIG_MODULE}_LINK_LIBRARIES}" STREQUAL "")
+    # Get and set the libraries and their absolute paths
+    list(APPEND link_libs ${${PKG_CONFIG_MODULE}_LINK_LIBRARIES})
   endif()
 
   hunter_status_debug(
-      "PKG_CONFIG_MODULE ${PKG_CONFIG_MODULE} LDFLAGS_OTHER: ${${PKG_CONFIG_MODULE}_LDFLAGS_OTHER}"
+      "PKG_CONFIG_MODULE ${PKG_CONFIG_MODULE} LIBRARY_DIRS: ${${PKG_CONFIG_MODULE}_LIBRARY_DIRS}"
   )
-  string(COMPARE NOTEQUAL
-      "${${PKG_CONFIG_MODULE}_LDFLAGS_OTHER}"
-      ""
-      has_ldflags_other
+  if(NOT "${${PKG_CONFIG_MODULE}_LIBRARY_DIRS}" STREQUAL "")
+    # Get only the paths of the libraries without '-L' and set with the '-L'
+    foreach(_dir ${${PKG_CONFIG_MODULE}_LIBRARY_DIRS})
```

Comment:
There is no need to start variable name with `_`, we are inside function. Can be simplified to:
```cmake
foreach(dir ${${PKG_CONFIG_MODULE}_LIBRARY_DIRS})
  list(APPEND link_libs -L${dir})
endforeach()
```

Reply:
Yep can be simplified ... but I added my second "fix request". So this line is changed and you will see why I cannot simplified.